client/src/App.js

```diff
@@ -22,9 +24,7 @@ class App extends Component {
           <hr/>
           <ItemPage/>
           <hr/>
-          <hr/>
-          <LoginModal/>
```

Comment:
`LoginModal` was breaking the app render. There's something wrong with it down the component stack. This is an example of "test before you merge"